### ISSUE TITLE
Bug: Quest Preview - losing _.numAdventurers() count

### ISSUE THREAD

**Issue by ArcanaIreanol** [ExpeditionRPG/expedition-quest-creator#435]
in a combat, I wanted to make it dynamic if there are more players, so I've written that:

```
adventurers: {{ _.numAdventurers() }}

Prepare for a fight

_combat_
- Archer
- Footpad
- Brigand
- Thief
- {{ _.numAdventurers() > 4 }} Highwayman
- {{ _.numAdventurers() > 5 }} Rogue
- QuarterMaster
* on win 
  congrats
  **end**
* on lose
  you s*ck
  **end**
* on round
  adventurers: {{ _.numAdventurers() }}
```

The issue is that when I preview from cursor, I then set the adventurers to 6 (settings), I then see before combat "adventurers: 6"
then combat begins, without the 2 extra foes
then on each round, I see "adventurers: 1"

Step 1: no config
![image](https://user-images.githubusercontent.com/32588637/32496346-3966a936-c3c9-11e7-9ddd-6bdbe8600e92.png)
Step 2: Configuring
![image](https://user-images.githubusercontent.com/32588637/32496369-4b92bdb6-c3c9-11e7-8b1f-d072d5feacd4.png)

Then I hit back and
Step 3: Play from cursor
![image](https://user-images.githubusercontent.com/32588637/32496444-864710a6-c3c9-11e7-806d-2dfaf8953d31.png)

Step 4: Here we see that we're now 6
![image](https://user-images.githubusercontent.com/32588637/32496461-941ba57a-c3c9-11e7-82e6-36681a7ab169.png)

Step 5: UH? where are my extra foes?
![image](https://user-images.githubusercontent.com/32588637/32496468-9bec5498-c3c9-11e7-94ad-7745fe5b393e.png)

timer, resolve and
Step 6: the * on round text
![image](https://user-images.githubusercontent.com/32588637/32496480-a73c486c-c3c9-11e7-8c87-2dee3b1a15e5.png)

BTW: click on top right button to get the settings opens a popover... but it's written white on yellow, hard to read

**Comment by ArcanaIreanol** [comment]
the only way to make it work is to F5

**Comment by toddmedema** [comment]
@ArcanaIreanol ah - so, hitting play from cursor reboots the app, and the quest creator currently sets the app to have one player. We have another issue here to make the app settings configurable: https://github.com/ExpeditionRPG/expedition-quest-creator/issues/261

However, note that we do not recommend changing combat based on the number of players, as the app dynamically changes combat difficulty based on the number of players. If you have consistently found that it's too easy with 5-6 players, we should file that as a separate issue to fix combat scaling, since we don't want to force every quest creator to have to individually balance every fight for every possible number of players :)

Good suggestion, I've added an issue here to add functions for alive adventurers and current tier: https://github.com/ExpeditionRPG/expedition-app/issues/517

**Comment by ArcanaIreanol** [comment]
Well, we played 6, but only once, didn't had the time to stress test. The issue was: it's going too fast. Even though the foes were doing nice dmg, we did massively more
Basically. When 6, we can combo a lot and deal massive dmg, therefore one shot a tier 4 with only 1 tier 1 loot used. Before they even begin

**Comment by toddmedema** [comment]
Gotcha. You might also consider either giving a bit more loot to small parties (to speed them up) or a bit less loot to large parties (to slow them down) so that you don't have to balance every fight individually.

**Comment by ArcanaIreanol** [comment]
Nice one. But:
-1- I'd make that automatic as well
-2- even though it balances, it reduces the satisfaction. As RPG player, we LOVE to receive loot (and as GM, we know players love that). And well... The main purpose of this game to me is to enjoy by having challenge, fun and satisfaction

**Comment by toddmedema** [comment]
1. Good point! Once we figure out some sort of function to say `rewardLoot(3)`, we could absolutely have that auto-balance.
2. Also a good point. So maybe rewarding more loot to small groups? Not only will they enjoy receiving the loot, but combat will go faster, which is a nice bonus since small group combat can go a bit slow.
